_docs/usecases/kubernetes/backups.md

```diff
@@ -0,0 +1,143 @@
+---
+layout: guide
+title: StorageOS Docs - Backups
+anchor: usecases
+module: usecases/kubernetes/backups
+---
+
+# File exfiltration on Kubernetes from StorageOS persistent volumes
+
+This example is provided to give ideas of how to exfiltrate files that have
+been written to a StorageOS persistent volume. This example uses nginx, rsync
+and sftp sidecars in an application pod. This approach may be useful to create
+out-of-cluster backups of data.
+
+In this example the "application" container is the container "main", which has
+rsync, nginx and sftp sidecar containers. The StorageOS volume that the
+application is writing to will be mounted into the sidecar containers as well so
+backup files etc. can be exfiltrated. 
+
+The files create a stateful set that can be used *AFTER* a StorageOS cluster
+has been created. For more information on how to install a StorageOS cluster
+please see [the StorageOS documentation]
+(https://docs.storageos.com/docs/introduction/quickstart).
+
+## Deploy
+
+In order to deploy the pod clone this repostiory and use kubectl to create the
+Kubernetes objects.
+
+> Before deploying the backup-example stateful set we highly recomend looking
+> through the examples to understand how the different containers are
+> configured 
+
+```bash
+$ git clone https://github.com/storageos/deploy.git storageos
+$ cd storageos
+$ kubectl create -f ./k8s/examples/backup-example
+```
+Check that a backup-example pod is running
+
+```bash
+$ kubectl get pods -w -l app=backup-example
+   NAME        READY    STATUS    RESTARTS    AGE
+   backup-example-0     1/1      Running    0          1m
+```
+
+## Exfiltrating files through Nginx
+
+Connect to the Nginx container running inside the backup-example pod through
```

Comment:
I think is not clear, what I read: "connect to the Nginx by execing into the busybox..."

I'd say:
We use `wget` from inside the Busybox Pod to show how to access the files served by the Nginx container (that shares the document root with the StorageOS volume location).

First ensure that the main Pod has written data. 
Second, check out the service endpoint created from `10-service.yaml`. 

`kubectl get svc`